### DISCUSSION TITLE
SetOnDropped functionallity when running app in a browser

### DISCUSSION THREAD

**Discussion by Matthijs-Movensis** [fyne-io/fyne#4678] [Q&A]
I made an app that uses the SetOnDropped functionallity. This works _very_ well on my Windows machine. However, I'd like to be able to deploy the app on the web. When testing using fyne serve, the app shows up, but the SetOnDropped functionallity doesn't work. Is there an alternative way to do this using fyne?

**Comment by andydotxyz** [comment]
The window handling on web driver is not 100% yet, this is one of the calls missing.
let's get this in for 2.5 along with the storage work that is under way.

**Comment by andydotxyz** [comment]
I have added issue #4679 to track this

**Comment by Matthijs-Movensis** [comment]
Thanks for the quick response, looking forward to use it in the future.
